Dockerfile

```diff
@@ -5,14 +5,14 @@ WORKDIR /go/src/github.com/gardener/gardener-extension-runtime-gvisor
 COPY . .
 RUN make install install-binaries
 ############# gardener-extension-runtime-gvisor
-FROM alpine:3.15.0 AS gardener-extension-runtime-gvisor
+FROM scratch AS gardener-extension-runtime-gvisor
```

Comment:
Can we keep this with alpine as well so that this is in line with all the other extensions?

Reply:
@kon-angelo @dkistner @ialidzhikov  would be good to have a response here

Reply:
I also agree. I am a fan of the `alpine` convenience but there is a tangible security benefit to using `scratch` that we should take into account. I am okay with going through with this change and try to adopt to other extensions.

Reply:
OK, thanks everybody. Then let's proceed.

> I am okay with going through with this change and try to adopt to other extensions.

Could you follow up on this then, please? Thanks!